jenkins-jobs/mysql-matrix-test.yaml

```diff
@@ -0,0 +1,51 @@
+# Job definition to test MySQL connector against different MySQL versions
```

Comment:
I'm not sure whether/where we CI this already, but can we add GTID on/off as an axis here, too?

Reply:
@gunnarmorling I've enabled `assembly` profile that should execute tests against multiple topologies.